source/monitoring/cluster-monitoring.txt

```diff
@@ -81,53 +83,79 @@ outputs messages such as the following:
 Event Descriptions
 ------------------
 
-You can subscribe to the following SDAM events by implementing the
-corresponding method from the ``SDAMSubscriber`` interface:
+You can subscribe to SDAM events by implementing the corresponding
+method from the ``SDAMSubscriber`` interface. The following table
+provides the name of each SDAM event, linked to the class's API
+documentation, and a description of when the event is published:
 
 .. list-table::
    :widths: 35 65 
    :header-rows: 1
 
-   * - Event Name
+   * - Event Type
      - Description
 
-   * - ``ServerChangedEvent``
-     - Created when an instance state changes, such as from secondary to
-       primary.
+   * - :php:`ServerChangedEvent <mongodb-driver-monitoring-serverchangedevent>`
+     - Created when the server description changes, such as the server's type
+       changing from secondary to primary.
 
-   * - ``ServerOpeningEvent``
-     - Created when the server is initialized.
+   * - :php:`ServerOpeningEvent <mongodb-driver-monitoring-serveropeningevent>`
+     - Created when the server description is instantiated with its
```

Comment:
For PHP, I think this can say "when a server connection is established". I'm asking for feedback on this in the drivers channel, but I don't think the spec's language about instantiating a server description is relevant to users.

For other drivers, I expect the public-facing docs would be "when the driver establishes a monitoring connection to the server".

Depending on the feedback I receive, perhaps we may want to say "immediately before the driver establishes a connection to the server" (or "monitoring connection to the server" for other drivers).

Reply:
@rustagir: Feedback came back. I think we're good to simply say "when a server connection is established".